### DISCUSSION TITLE
✅ 🔗 Download Google Font to local Custom Fonts (Duplicate of #19966)

### DISCUSSION THREAD

**Discussion by VerdiH** [elementor/elementor#21716] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Issues with GDPR. Slower website.

### Describe the solution you'd like

We can already use custom fonts and disable Google fonts altogether. Why not add an option to import them to Custom Fonts like other plugins are already doing? This would save time, prevent certain GDPR issues and result in better optimized websites.

As the Google fonts are Open Source, it seems to be allowed. 

### Describe alternatives you've considered

Using plugins like this https://wordpress.org/plugins/local-google-fonts/ or this https://webfont.yabe.land/ but I do not want yet another plugin / blackbox in my sites website.

I've also been looking into variable fonts. That would be even better.

**Comment by awebguy99** [comment]
Great idea.

**Comment by nicholaszein** [comment]
Hey @VerdiH 🙂!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #19966

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hi @VerdiH! I'm sure you know this (😅), but doesn't hurt to say:

We have great news! 🙌 

### 📢 We're happy to announce that Load Google Fonts Locally is now a native feature of Elementor since the release of version 3.27.0! 🥳 

You can now activate the new experimental feature by going to your WordPress **`Dashboard`** > **`Elementor`** > **`Settings`** > **`Features`** page, and switching the option to **`Load Google Fonts locally`** to **`Active`**.

This will make Elementor automatically download all the weights and style files of the Google Fonts in use in the **`.woff2`**, and save them in a special folder inside the **`../wp-content/uploads/`** folder.

These font files are downloaded only once a page using the Google Fonts is loaded, and will be re-downloaded on every new visit after the **`Elementor CSS & Data`** is **`Regenerated`**. Fonts are only loaded when in use.

For more information see: https://elementor.com/help/load-google-fonts-locally/

> _In the future this feature will be merged to core and may or may not have a setting in the **`Elementor Settings`** > **`Performance`** tab._

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
